README.md

```diff
@@ -274,6 +277,29 @@ out the necessary python commands to run to deactivate your env and remove the c
 
 ```shell script
 make teardown
+### Setup: Nix
```

Comment:
Looks like the trailing three ` that should go after `make teardown` are missing which is messing with the format.

Reply:
Thanks! I think I fixed that